karma.conf.js

```diff
@@ -1,22 +1,72 @@
+// Karma configuration
+// Generated on Wed Dec 06 2017 14:07:02 GMT-0600 (CST)
+
 module.exports = function(config) {
   config.set({
+
+    // base path that will be used to resolve all patterns (eg. files, exclude)
+    basePath: '',
+
+
+    // frameworks to use
+    // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
     frameworks: ['mocha'],
 
+
+    // list of files / patterns to load in the browser
     files: [
-      '*.js'
+      './client/**/**/*.spec.js'
```

Comment:
Karma runs test in a web browser, so it only makes sense to include the client specs.